test/api/buffer/exports/delete/expected-state.json

```diff
@@ -1,55 +0,0 @@
-{
-  "branches": [
-    {
```

Comment:
`exports/delete` test casto pada na tom, ze Storage API vrati zly pocet suborov, uz som tam dal aj delay `100ms`, ale nepomohlo to.

Tak som z tohto testu vyhodil kontrolu stavu, nie je tam potrebna.
V inych testoch je to lepsie, kedze trvaju dlhsie od vytvorenia suboru.